tools/storage-explorer/go.mod

```diff
@@ -4,7 +4,7 @@ go 1.20
 
 require (
 	github.com/gorilla/mux v1.8.1
-	github.com/onflow/atree v0.6.1-0.20240314011440-92714cac03fb
+	github.com/onflow/atree v0.6.1-0.20240417191405-e11f55fa33a3
```

Comment:
The main go.mod file on the atree register inlining is the same, but this other go.mod file was forgotten to also be updated